### DISCUSSION TITLE
请问如何指定package的安装目录？

### DISCUSSION THREAD

**Discussion by wanzzhehe** [xmake-io/xmake#3705] [Q&A]
希望直接把包安装在指定的目录下，例如 "$(projectdir)/build/packages"，然后优先从这个目录下查找安装包，请问怎么实现？

**Comment by waruqi** [comment]
https://xmake.io/#/zh-cn/guide/configuration?id=xmake_pkg_installdir

**Reply by wanzzhehe** [reply]
那请问有没有只在当前工程内生效的配置呢……
